src/components/socials.js

```diff
@@ -28,17 +29,36 @@ const SocialLink = styled(Link)`
 `;
 
 const Socials = () => {
+  const data = useStaticQuery(graphql`
+    query socialQuery {
+      file(
+        relativePath: { eq: "socials.md" }
+        sourceInstanceName: { eq: "markdown" }
+      ) {
+        childMarkdownRemark {
+          frontmatter {
```

Comment:
Do you want to do the weird renaming thing that @khanguslee spoke about yesterday?

Reply:
I looked into that and I cant find a way to do it in Gatsby. I think you have to traverse the whole JSON object to find the data you want.